admin_guide/out_of_resource_handling.adoc

```diff
@@ -116,21 +119,20 @@ To configure hard eviction thresholds, the following flag is supported:
 The node evaluates eviction thresholds per its housekeeping interval. It
```

Comment:
As noted earlier, I think we should just say:

"The node evaluates eviction thresholds every 10s."

In the future, hard eviction thresholds for memory will not use polling every 10s, and instead we will have the kernel tell us the threshold has been passed and act immediately.  That is planned for Kubernetes 1.5 / Origin 1.5.